pkg/policy/consumer.go

```diff
@@ -258,37 +246,36 @@ func (c *Consumable) AllowConsumerAndReverseLocked(cache *ConsumableCache, id Nu
 		}).Debug("Allowing reverse direction")
 		if _, ok := reverse.ReverseRules[c.ID]; !ok {
 			reverse.addToMaps(c.ID)
-			reverse.ReverseRules[c.ID] = NewConsumer(c.ID)
+			reverse.ReverseRules[c.ID] = true
 			return true
 		}
 	}
 	log.WithFields(logrus.Fields{
 		logfields.Identity + ".from": c.ID,
 		logfields.Identity + ".to":   id,
-	}).Warn("Allowed a consumer which can't be found in the reverse direction")
+	}).Warn("Allowed an ingress security identity which can't be found in the reverse direction")
 	return changed
 }
 
-// BanConsumerLocked removes the given consumer from the Consumable's consumers
-// map. Must be called with the Consumable mutex locked.
-func (c *Consumable) BanConsumerLocked(id NumericIdentity) {
-	if consumer, ok := c.Consumers[id.StringID()]; ok {
-		log.WithField("consumer", logfields.Repr(consumer)).Debug("Removing consumer")
-		delete(c.Consumers, id.StringID())
+// RemoveIngressIdentityLocked removes the given security identity from Consumable's
+// IngressIdentities map.
+// Must be called with the Consumable mutex locked.
+func (c *Consumable) RemoveIngressIdentityLocked(id NumericIdentity) {
+	if _, ok := c.IngressIdentities[id]; ok {
+		log.WithField(logfields.Identity, id).Debug("Removing ingress identity")
```

Comment:
Replying to this comment in response to your review - I agree 100%, this might be confusing to end-users. Will fix such messages in my next upload :-)

Reply:
Done.